### PR TITLE
All websites dashboard lists website names double encoded

### PR DESCRIPTION
**Pull request by tsteur**
refs #7806 

Should be XSS safe. We do HTML decode in the browser but when outputting it in the browser, it will be encoded again by AngularJS

### CODE REVIEW

File: tests/UI/specs/MultiSites_spec.js [hunk 1]
```diff
@@ -13,12 +13,38 @@ describe("MultiSitesTest", function () {
     var generalParams = 'idSite=1&period=year&date=2012-08-09';
     var selector = '.pageWrap,.expandDataTableFooterDrawer';
 
+    var createdSiteId = null;
+
+    before(function (done) {
+        var callback = function (undefined, response) {
```
Comment (diosmosis) [inline tests/UI/specs/MultiSites_spec.js]: The first argument to the callback will be an error (ie, it's a node.js style callback). If defined, you should call `done(error)` (where error is `undefined` renamed).

Comment (tsteur) [inline tests/UI/specs/MultiSites_spec.js]: I'm not sure if I understand.

Should I do something like this?
```js
var callback = function (error, response) {
// ...
 done(error, response)
}
```

Comment (diosmosis) [inline tests/UI/specs/MultiSites_spec.js]: More like:

```
var callback = function (error, response) {
    if (error) {
        done(error);
        return;
    }

    ...

    done();
}
```

See: http://thenodeway.io/posts/understanding-error-first-callbacks/

Comment (tsteur) [inline tests/UI/specs/MultiSites_spec.js]: cool, done

**Pull request merged**